projects/jettison/project.yaml

```diff
@@ -0,0 +1,12 @@
+homepage: "https://github.com/jettison-json/jettison"
```

Comment:
Do any of the upstream maintainers want this?

Reply:
I have not heard back from them yet.